src/bitcoin-util.cpp

```diff
@@ -60,7 +60,9 @@ static int AppInitUtil(ArgsManager& args, int argc, char* argv[])
             strUsage += FormatParagraph(LicenseInfo());
         } else {
             strUsage += "\n"
-                "Usage:  bitcoin-util [options] [commands]  Do stuff\n";
+                "The bitcoin-util tool is used to grind proof of work on a hex-formatted bitcoin block header.\n"
```

Comment:
```suggestion
                "The bitcoin-util tool provides bitcoin related functionality that does not rely on the ability to access a running node.\n"
```

Why not use the description from #19937 which introduced the tool?

Reply:
Done